modules/nw-multus-delete-network.adoc

```diff
@@ -11,7 +11,7 @@ is attached to.
 
 [IMPORTANT]
 ====
-In {product-title} 4.2.0 you must manually delete the additional network CR after removing it from the Cluster Network Operator configuration.
+In {product-title} 4{product-version}, you must manually delete the additional network CR after removing it from the Cluster Network Operator configuration.
```

Comment:
@ahardin-rh this looks to be a bad search and replace?

@jboxman is it correct to replace this here with product-version instead of 4.2? Does this sentence still apply for 4.3?

Reply:
@vikram-redhat Oops! Fixed. Thanks!

Reply:
@jboxman bump

Reply:
@jboxman Can you PTAL when you have a moment?

Reply:
Got a :+1: from @jboxman in Slack.